source/features/latest-tag-button.tsx

```diff
@@ -136,7 +136,7 @@ async function init(): Promise<false | void> {
 				<a
 					className="btn btn-sm btn-outline tooltipped tooltipped-ne"
 					href={buildRepoURL(`compare/${latestTag}...${defaultBranch}`)}
-					data-pjax="#repo-content-pjax-container"
+					data-pjax="#js-repo-pjax-container"
```

Comment:
No, this won't work either

https://user-images.githubusercontent.com/44045911/128203855-b5f5b3dd-437c-4fc8-a905-ac4976f91517.mov

Reply:
Okay maybe I shouldn't do quick fixes for this 😅
But interesting that the next navigation is the issue in that case.